Task_1/Task_1/src/components/Input_form/form.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+
+const Form = (props) => {
+  const [enterTask, setEnterTask] = useState("");
+  const [enterDate, setEnterDate] = useState("");
+
+  function TaskHandler(event) {
+    setEnterTask(event.target.value);
+  }
+  function DateHandler(event) {
+    let date = event.target.value.toString();
+    setEnterDate(date);
+  }
+
+  function add_task() {
+    if(enterTask=="" || enterDate == ""){
+      console.log("not valid title")
+      return;
+    }
+    let flag_duplicate_item = false;
+    props.list.map((obj)=>{
+      if(obj.title == enterTask){
+        flag_duplicate_item = true;
+      }
+    })
+    if(flag_duplicate_item){
+      return;
+    }
+    console.log("button is worling");
+    console.log(enterTask);
+    props.add_task(enterTask, enterDate);
+  }
+
+  function formSubmitHandler(e){
+    e.preventDefault();
+    document.getElementById("formI").reset();
```

Comment:
not a correct way of reset. avoid using this React @codal-uchoudhary. instead use setState method to bring back it to it's initial state.